configuration/introduction.md

```diff
@@ -93,15 +165,110 @@ MY_HTTPS=false
 
 ##### Using linux environment variables
 
-You have to declare them before you start MagicMirror², e.g. by executing:
+define them before you start MagicMirror², in a bash script, for example: 
 
-```shell
+```bash
+cd ~/MagicMirror
 export MY_ADDRESS=localhost
 export MYPORT=8080
 export MY_HTTPS=false
+npm start
+```
+
+
+
+#### Using `electronOptions`: 
+
+The most common use for `electronOptions` is a dual monitor setup on a Raspberry Pi 4 or greater. To do so, you can launch a second MagicMirror and use `electronOptions` to move it over to the next monitor. For example:
+
+You have two 1920x1080 monitors set up in the Pi as sitting next to each other: 
+![image-20231226175637817](dualmon.png)
+
+
+
+To move the pi to the second monitor, use this: 
+
+```js
+var config = {
+	electronOptions: { x: 1920 },
+	...
+```
+
+
+
+#### A Couple of Real World Examples: 
+
+##### Two Screens: 
+
+A user has two monitors running on a Pi4B.  User wants to show cameras on one monitor, and wants to show informational panels on the other. Monitors are set up as above. 
+
+User would have to set up two config files, and two starting scripts to do so. User would use method of choice to launch starting scripts (e.g.: `PM2`). 
+
+Starting Script 1 (mm.sh): 
+
+```bash
+cd ~/MagicMirror
+npm start
 ```
 
-#### Real world example of `config.js.template`
+Starting Script 2 (mm2.sh): 
+
+```bash
+cd ~/MagicMirror
+export MM_CONFIG_FILE=${pwd}/config/config2.js
```

Comment:
can you please omit the `${pwd}/` part here?

Reply:
Fixed.